trade-finance-manager-api/src/graphql/resolvers/query-facilities.js

```diff
@@ -19,15 +19,23 @@ exports.queryAllFacilities = async (queryParams) => {
     // finds latest completed amendment tfm object with mapped values
     const latestCompletedAmendment = findLatestCompletedAmendment(item?.amendments);
 
+    let facilityCoverEndDate = '-';
```

Comment:
Is there a reason why the variable `facilityCoverEndDate` has been instantiated with `-`?

Reply:
fixed